dockertls/README.md

```diff
@@ -63,6 +63,34 @@ $env:DOCKER_TLS_VERIFY="1"
 docker version
 ```
 
+## Managing Multiple Hosts
+
+## First Host
+For the first host you must create an empty directory to hold the Certificate Authority Private Keys.
+Ideally the key and password should be kept separate and only provided when additional certificates are created.
+```
+mkdir $env:SystemDrive\DockerSSLCARoot
+mkdir $env:USERPROFILE\.docker
+docker run --rm `
+  -e SERVER_NAME=$(hostname) `
+  -e IP_ADDRESSES=127.0.0.1,192.168.254.135 `
```

Comment:
Yes, `192.168.254.135` stands for an external/public IP address.

Here is a better example to add external and all IP addresses:

```powershell
$externalIp = "10.100.xx.xx" # external edge gateway

if (!(Test-Path $env:USERPROFILE\.docker)) {
  mkdir $env:USERPROFILE\.docker
}

$ips = ((Get-NetIPAddress -AddressFamily IPv4).IPAddress) -Join ','
Write-Host "Creating certs for $ips,$externalIp"

docker run --rm `
  -e SERVER_NAME=$(hostname) `
  -e IP_ADDRESSES=$ips,$externalIp `
...
```

I think the readme should be improved a little bit :-)